packages/embark/src/lib/modules/blockchain/index.js

```diff
@@ -20,18 +21,31 @@ class Blockchain {
       this.blockchainNodes[clientName] = startCb;
     });
 
-    this.events.setCommandHandler("blockchain:node:start", (blockchainConfig, cb) => {
-      const clientName = blockchainConfig.client;
-      // const clientName = this.blockchainConfig.client;
-      const client = this.blockchainNodes[clientName];
-      if (!client) return cb("client " + clientName + " not found");
+    this.events.setCommandHandler("blockchain:node:start", async (blockchainConfig, cb) => {
+      const requestManager = new Web3RequestManager.Manager(blockchainConfig.endpoint);
 
-      let onStart = () => {
-        this.events.emit("blockchain:started", clientName);
-        cb();
-      };
+      const ogConsoleError = console.error;
+      // TODO remove this once we update to web3 2.0
+      // TODO in web3 1.0, it console.errors "connection not open on send()" even if we catch the error
```

Comment:
This comment concerns me. Are you saying we're overriding `console.error` to a noop functions so we avoid seeing that error message, because, for some reason, it's output even though "we catch the error"?

Maybe you can elaborate on what exactly is going on, this seems like a bad hack to me. Wouldn't be surprised if this could potentially affect other parts of the code base that rely on `console.error`

Reply:
Yeah pretty much. Long story short, there is this `console.error` in web3 1.0: https://github.com/ethereum/web3.js/blob/1.x/packages/web3-providers-ws/src/index.js#L276

They got rid of it in web3 2.0, but there is no way for it to not display right now.

It's very annoying for my use case, because I use web3 to test the connection, but it will display `connection not open on send()`, even though we will start the node afterwards.

But I just got an idea. I won't make `console.error` a noop, I'll instead filter out the message while waiting for the response and then I reestablish the right function like before.

Reply:
Fixed here: https://github.com/embark-framework/embark/pull/1874/commits/c76c2859b564448fa18162b74bd4fbb8fe56b9cd